package.json

```diff
@@ -14,6 +14,7 @@
     "build-dev": "bash ./build.sh dev",
     "docker:build": "bash ./docker-build-image.sh dev",
     "docker:run": "bash ./docker-run-image.sh dev",
+    "docker:start": "bash -c 'cat /KTH_NODEJS; NODE_ENV=development node app.js'",
```

Comment:
I'm not sure about the naming here. The only difference between "docker:start" and just "start" is that NODE_ENV is development instead of production. I was a little confused about this, because there is nothing docker-related by this at all. I understand that you are using this in a dockerfile, but the regular  "start" is also use in a dockerfile. 

My first thought was to just rename this one to "start-dev", but then I realised we already have a "start-dev" for other purposes.  

**But** maybe the bigger question maybe is: Why do we need to start it in dev mode? I think that one of the big advantages of running in docker here would be to test it as close as possible to a real "production" environment. What benefits do we get from starting with node_env=dev here?

Reply:
> I'm not sure about the naming here. The only difference between "docker:start" and just "start" is that NODE_ENV is development instead of production. I was a little confused about this, because there is nothing docker-related by this at all. I understand that you are using this in a dockerfile, but the regular "start" is also use in a dockerfile.
> 
> My first thought was to just rename this one to "start-dev", but then I realised we already have a "start-dev" for other purposes.
> 
> **But** maybe the bigger question maybe is: Why do we need to start it in dev mode? I think that one of the big advantages of running in docker here would be to test it as close as possible to a real "production" environment. What benefits do we get from starting with node_env=dev here?

It is great that you mentioned it, 
I shared my thoughts in the other comment regarding the dev mode for running docker.

regarding the new script that I have added I was thinking to make a script only for docker in dev mode but then I thought using start-dev would be better than creating a new one.

Thanks for your comment ⭐️